nmap.aliases.bash

```diff
@@ -0,0 +1,15 @@
+#Niil78
+#2019
+cite 'about-alias'
+about-alias 'Nmap scanner'
+
+alias Nmap0='echo "Escaner normal"; sudo nmap' #Escaner Basico
```

Comment:
Can you please convert the comments and `echo` output to English language? We have an international audience with Bash-it...

Reply:
If I'm going to do the whole sheet. I want to do a meno like in pacman.
:D